src/integration/pyboy_integration.py

```diff
@@ -28,4 +32,128 @@ def pyboy_tick(button=""):
     pyboy.screen.image.resize((480, 432), 0).save("data/image.png")
     with open("data/state_file.state", "wb") as f:
         pyboy.save_state(f)
+
+    game_info = GameInformation.from_pyboy(pyboy)
+
     pyboy.stop()
+
+    return game_info
+
+
+@dataclass
+class Pokemon:
```

Comment:
I wonder if we should be moving this in to a new `gen_1_integration.py` file, to keep it separate from generic pyboy stuff

Reply:
done